velox/exec/tests/UnnestTest.cpp

```diff
@@ -474,6 +474,40 @@ TEST_P(UnnestTest, batchSize) {
   ASSERT_EQ(expectedNumVectors, stats.at(unnestId).outputVectors);
 }
 
+TEST_P(UnnestTest, basicArrayWithOuter) {
+  auto vector = makeRowVector({
+      makeFlatVector<int64_t>({1, 2, 3}),
+      makeNullableArrayVector<int64_t>({{1, 2, std::nullopt}, {}, {3}}),
+  });
+
+  createDuckDbTable({vector});
+
+  // isOuter = true.
+  auto op2 = PlanBuilder()
```

Comment:
nit: can we rename it as op and reuse it at L498? Same for params2 and expected2.